<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>892</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>892</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>892</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MMwaveObserver</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QCustomPlot" name="mmwaveAdValuePlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>340</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>波形控制</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="automaticShiftCheckBox">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>自动平移</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="automaticZoomCheckBox">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>自动缩放</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>回到原点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>缩放控制</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6"/>
        </item>
        <item>
         <widget class="QRadioButton" name="zoomXRadioButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X轴缩放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="zoomYRadioButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y轴缩放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="zoomBothRadioButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>等比缩放</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7"/>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>播放速度</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QDial" name="displaySpeedDial"/>
          </item>
          <item>
           <widget class="QSpinBox" name="displaySpeedSpinBox"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>毫秒/点</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="Line" name="line_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>文件流控制</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QRadioButton" name="inputSelectFileRadioButton">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>文件输入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="inputSelectUartRadioButton">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>串口输入</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="inputSelectEthRadioButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>以太网输入</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9"/>
        </item>
        <item>
         <widget class="QPushButton" name="serialSettingPushButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>串口设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ethSettingPushButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>网络设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startPlotPushButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>开始绘制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pausePlotPushButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>暂停绘制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="replotPushButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>重新绘制</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopPlotPushButton">
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>停止绘制</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="openPlotPushButton">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>打开曲线</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="openFileFormatComboBox">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>raw</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>csv</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="savePlotPushButton">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>保存曲线</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="saveFileFormatComboBox">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>csv</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>raw</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>频域数据</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>原始数据</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QPushButton" name="newConfigPushButton">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>新建配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importConfigPushButton">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>导入配置</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveConfigPushButton">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>保存配置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" rowspan="4">
     <widget class="Line" name="line_13">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="Line" name="line_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QProgressBar" name="displayProgressBar">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QProgressBar {
     border: 2px solid grey;
     border-radius: 5px;
     text-align: center;
 }
 QProgressBar::chunk {
     background-color: #CD96CD;
     width: 10px;
     margin: 0.5px;
 }</string>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="6" column="2" rowspan="3">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>波形处理</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dspLPFCheckBox">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>低通滤波</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="dspBFPCheckBox">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>带通滤波</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_12">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>波形信息</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>呼吸率(次/分)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>心率(次/分)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLCDNumber" name="breathRateLcdNumber">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>215</width>
              <height>102</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>20</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="heartRateLcdNumber">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>215</width>
              <height>102</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>9</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13"/>
      </item>
      <item>
       <widget class="QLabel" name="hduLogoLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <widget class="QCustomPlot" name="mmwaveProcessedPlot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>320</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
